docs/Compiler-Hardening-Guides/Compiler-Options-Hardening-Guide-for-C-and-C++.md

```diff
@@ -25,6 +25,18 @@ When compiling C or C++ code on compilers such as GCC and clang, turn on these f
 -fPIE -pie -fPIC -shared
 ~~~~
 
+When compiling on x86 architectures, add:
```

Comment:
`When compiling on x86 (amd64, i386 and x32) architectures, add:`
not all x86 is supported

Reply:
I'm not sure what to do here. Does anyone have a recommendation? I plan to fix the rest & leave this open, I'm not sure I understand the real problem in this case.

Reply:
`When compiling on most x86 architectures (amd64, i386 and x32), add:`

@siddhesh does that sound okay to you?

Reply:
Sure, LGTM.

Reply:
Ok, I'll do that.